### PR TITLE
Release 0.9.1

### PR DESCRIPTION
**Pull request by basetenbot**
Updating Truss from ["0.9.0"](https://pypi.org/pypi/truss/json) to 0.9.1. **PLEASE ENSURE YOU MERGE, NOT SQUASH**

### CODE REVIEW

File: truss/contexts/image_builder/util.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 # This needs to be updated whenever we want to update
 # base images on a merge. Updating this version will cause
 # base images to be pushed with this tag.
-TRUSS_BASE_IMAGE_VERSION_TAG = "v0.7.17"
+TRUSS_BASE_IMAGE_VERSION_TAG = "v0.9.0"
```
[the remark this answers is not in the document]
Comment (squidarth) [reply truss/contexts/image_builder/util.py:9]: yep, they are there: https://hub.docker.com/layers/baseten/truss-server-base/3.10-gpu-v0.9.0/images/sha256-339303575d7ec7076550a53fa6e4e183b0cf72a66e0666e26588bb87e3c6d141?context=explore

File: truss/templates/control/control/endpoints.py [hunk 2]
```diff
@@ -109,7 +109,9 @@ async def patch(request: Request) -> Dict[str, str]:
 
 @control_app.get("/control/truss_hash")
 async def truss_hash(request: Request) -> Dict[str, Any]:
+    request.app.state.logger.info(f"truss hash request received. request: {request}")
```
[the remark this answers is not in the document]
Comment (helenlyang) [reply truss/templates/control/control/endpoints.py:112]: Yeah, I'll open a revert for this if it's ok to hold off on release for a bit. Showing these logs to users is not ideal

[the remark this answers is not in the document]
Comment (helenlyang) [reply truss/templates/control/control/endpoints.py:112]: https://github.com/basetenlabs/truss/pull/832 has been merged. Is there an easy way to pull it into this PR or do we need to create a new release PR?

[the remark this answers is not in the document]
Comment (bolasim) [reply truss/templates/control/control/endpoints.py:112]: you can cherrypick into this PR, but it might be just cleaner to make a new release PR

**Comment by helenlyang** [comment]
Closing this, will reopen a new release PR with #832 ([thread](https://github.com/basetenlabs/truss/pull/831#discussion_r1499585744))

**Pull request closed without merging**